Possible jshint bug, not commenting missing semicolon

Ran into this- locally I have missing semicolon issue:

λ npm test
> jshint . && jscs .

commands.js: line 295, col 87, Missing semicolon.
commands.js: line 554, col 2, Missing semicolon.

2 errors
npm ERR! Test failed.  See above for more details.

--- a/inlineplz/linters/__init__.py
+++ b/inlineplz/linters/__init__.py
@@ -32,7 +32,7 @@ LINTERS = {
     },
     'jshint': {
         'install': ['npm', 'install', 'jshint'],
-        'run': [os.path.normpath('./node_modules/.bin/jshint'), '--reporter', 'checkstyle'],
+        'run': [os.path.normpath('./node_modules/.bin/jshint'), '.', '--reporter', 'checkstyle'],
         'dotfiles': ['.jshintrc'],
         'parser': parsers.JSHintParser
     },

--- a/inlineplz/parsers/jshint.py
+++ b/inlineplz/parsers/jshint.py
@@ -12,13 +12,14 @@ class JSHintParser(ParserBase):
     def parse(self, lint_data):
         messages = set()
         obj = xmltodict.parse(lint_data)
-        for filedata in obj['checkstyle']['file']:
-            for errordata in filedata['error']:
-                try:
-                    path = filedata['@name']
-                    line = int(errordata['@line'])
-                    msgbody = errordata['@message']
-                    messages.add((path, line, msgbody))
-                except (AttributeError, TypeError):
-                    pass
+        if 'file' in obj['checkstyle']:
+            for filedata in obj['checkstyle']['file']:
+                for errordata in filedata['error']:
+                    try:
+                        path = filedata['@name']
+                        line = int(errordata['@line'])
+                        msgbody = errordata['@message']
+                        messages.add((path, line, msgbody))
+                    except (AttributeError, TypeError):
+                        pass
         return messages

